index.cjs

```diff
@@ -4,7 +4,12 @@
 var path = require("path");
 
 // If in 'production' mode, use the combined/minified/optimized version of Cesium
-if (process.env.NODE_ENV === "production") {
+// PROPELLER HACK: Use minified code in all deployed lambda environments for compactness
```

Comment:
This will probably affect visualiser releases too right?  Do we want to retain unminified code there?

Reply:
I am undoing this, unintentional

Reply:
Restored from master